app/dao/organisation_dao.py

```diff
@@ -26,11 +34,22 @@ def dao_get_organisation_services(organisation_id):
     ).one().services
 
 
-def dao_get_organisation_live_services(organisation_id):
-    return Service.query.filter_by(
-        organisation_id=organisation_id,
-        restricted=False
-    ).all()
+def dao_get_organisation_live_services_with_free_allowance(organisation_id, financial_year):
```

Comment:
Optional: I initially read 'with_free_allowance' as a filter. Ie only get services which have a free allowance and exclude those that don't have any free allowance.

Not sure if it is worth changing to something like 'dao_get_organisation_live_services_and_their_free_allowance'